systemd/ircbot.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Schalter IRC Bot
+
+[Service] 
+ExecStart=/usr/bin/sopel
+User=pi
+Restart=always
+RestartSec=3
```

Comment:
Eventuell wollt ihr noch folgendes hinzufügen:

```
StartLimitIntervalSec=30
StartLimitBurst=3
```

Oder sowas. Dann wird die Unit deaktiviert, wenn der Service innerhalb von ~21 Sekunden mehr als drei mal abstürzt. (21, weil 30-3*3 wegen `RestartSec`)

Reply:
Das wäre jedenfalls nett gegenüber dem IRC-Server.

Reply:
sopel ist ein completer daemon - der hat reconnect und co - der Fehler in eigenen Modulen abfängt.

Zudem sind diese Werte vom Betriebsystem mit Default-Werten gesetzt siehe (`DefaultStartLimitBurst`).
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#StartLimitIntervalSec=interval

Reply:
Die Defaults sind bei mir wie folgt:

```
#DefaultStartLimitIntervalSec=10s
#DefaultStartLimitBurst=5
```

D.h. dieser Service würde da nie hereinlaufen, weil er drei Sekunden Restart-Delay hat.

Es ist schön, dass sopel reconnect-Fähigkeiten hat, aber das kann auch alles mal falsch implementiert sein und dann wäre es nett, nicht für alle Ewigkeit alle drei Sekunden zu Hackint zu verbinden.

Reply:
Habs auf 5min bot und 30s schalter service geändert - okay so?

Reply:
Ich bin mir nicht ganz sicher, was du meinst...